test/starlark_tests/resources/Info-nopackage.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`Info-nopackage.plist` was added for `macos_dylib` and `macos_command_line_application`. When using Info.plist embedding in binaries, Xcode will completely omit the `CFBundlePackageType` key, even if you include it in your plist (presumably because neither are bundles)

Some logic should be added to the plist tooling to support omitting keys for Info.plist in the embedded use case (this was a bit out-of-scope for this PR specifically)